.env.template

```diff
@@ -23,3 +23,7 @@ ETHEREUM_SUB_THREAD_CNT=
 PORT=
 NETUID=
 NETWORK=
+
```

Comment:
general comment: we are creating here new model, so we should put those into
bitcoin/account_balances (or sth like that)

and for general requirement, miners should be running both models